include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -65,68 +65,67 @@ namespace seqan3
  */
 /*!\name Requirements for seqan3::aligned_sequence_concept
  * \brief You can expect these functions on all types that model the seqan3::aligned_sequence_concept.
+ * \relates seqan3::aligned_sequence_concept
  *
  * \details
  *
  * Type Requirements:
- *   1. The type must model the std::ranges::ForwardRange.
- *   2. The `value_type` must model the alphabet concept.
- *   3. The `value_type` must be assignable from seqan3::gap::GAP.
+ *   1. The type must model std::ranges::ForwardRange.
+ *   2. The `reference type` must model seqan3::alphabet_concept.
+ *   3. The `reference type` must be assignable from seqan3::gap.
```

Comment:
there is no `reference_type`, it's just called `reference` ( I know it's inconsistent with "value_type").